zerver/openapi/openapi.py

```diff
@@ -218,6 +218,19 @@ def get_openapi_summary(endpoint: str, method: str) -> str:
     return openapi_spec.openapi()["paths"][endpoint][method.lower()]["summary"]
 
 
+def get_endpoint_from_operationid(operationid: str) -> str:
+    print(operationid)
+    for each_endpoint in openapi_spec.openapi()["paths"]:
+        for each_method in openapi_spec.openapi()["paths"][each_endpoint]:
```

Comment:
I'd call these local variables `endpoint` and `method`; no need for the `each_` prefix.